src/app/components/ATIAnalytics/params/genericPage/buildParams.ts

```diff
@@ -49,6 +50,7 @@ export const buildPageATIParams = ({
     statsDestination,
     timePublished,
     timeUpdated,
+    ...(ampExperimentName && { ampExperimentName }),
```

Comment:
**nitpick**: Is spreading needed here? If `ampExperimentName` is undefined, it won't be returned?

Reply:
If I left it as is in the object, the key `ampExperimentName` would still be there even if the value was falsey. As it is currently written, it won't work without the spread operator because it wouldn't have a valid key/pair if it evaluated as false.